artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/JsonUtil.java

```diff
@@ -16,13 +16,13 @@
  */
 package org.apache.activemq.artemis.api.core;
 
-import javax.json.JsonArray;
-import javax.json.JsonArrayBuilder;
-import javax.json.JsonNumber;
-import javax.json.JsonObject;
-import javax.json.JsonObjectBuilder;
-import javax.json.JsonString;
-import javax.json.JsonValue;
+import org.apache.activemq.artemis.commons.json.JsonArray;
+import org.apache.activemq.artemis.commons.json.JsonArrayBuilder;
+import org.apache.activemq.artemis.commons.json.JsonNumber;
+import org.apache.activemq.artemis.commons.json.JsonObject;
+import org.apache.activemq.artemis.commons.json.JsonObjectBuilder;
+import org.apache.activemq.artemis.commons.json.JsonString;
+import org.apache.activemq.artemis.commons.json.JsonValue;
```

Comment:
Just to be sure its intended/realised...noting this is presumably a breaking change for all existing management related usage of these bits. EDIT: As the later test changes show.

Reply:
Json is only used internally... we never (should not) expose through the API.

Reply:
Fair enough, just checking that was the position for when anyone comes asking about it.

Reply:
Another consideration in that case might be to remove (or move to a clearly-internal package) this class if its not considered API (which might seem odd given its in an "api" sub package)